script/upgrades/eXOF/Config.sol

```diff
@@ -60,32 +62,37 @@ library eXOFConfig {
       enabled: true,
       threshold: FixidityLib.newFixedFraction(3, 100), // 0.03
       cooldown: 30 minutes,
-      smoothingFactor: 0
+      smoothingFactor: 1e24 // 1
     });
-    rateFeedConfig.dependentRateFeeds = Arrays.addresses(contracts.dependency("EURXOFRateFeedAddr"));
+    rateFeedConfig.dependentRateFeeds = Arrays.addresses(Config.rateFeedID("EURXOF"));
   }
 
   /**
    * @dev Returns the configuration for the EURXOF rate feed.
    */
-  function EURXOF_RateFeedConfig(
+  function EUROCXOF_RateFeedConfig(
     Contracts.Cache storage contracts
   ) internal returns (Config.RateFeed memory rateFeedConfig) {
-    rateFeedConfig.rateFeedID = contracts.dependency("EURXOFRateFeedAddr");
+    rateFeedConfig.rateFeedID = Config.rateFeedID("EUROCXOF");
     rateFeedConfig.valueDeltaBreaker0 = Config.ValueDeltaBreaker({
       enabled: true,
-      threshold: FixidityLib.newFixedFraction(5, 1000), // 0.005
+      threshold: FixidityLib.newFixedFraction(10, 1000), // 0.01
```

Comment:
Why did we changed this threshold from `0.005` to `0.01`?

Reply:
I would also vote for `0.005`

Reply:
updated